packages/components/src/public/QueryModel/utils.ts

```diff
@@ -108,15 +108,15 @@ export function getQueryModelExportParams(
     type: EXPORT_TYPES,
     advancedOptions?: Record<string, any>
 ): Record<string, any> {
-    const { containerFilter, containerPath, exportColumnString, filters, hasSelections, id, schemaQuery, sortString } =
+    const { containerFilter, containerPath, exportColumnString, filters, hasSelections, schemaQuery, sortString, selectionKey } =
         model;
     const exportOptions: ExportOptions = {
         filters: List(filters),
         columns: exportColumnString,
         containerFilter,
         containerPath,
         sorts: sortString,
-        selectionKey: id,
+        selectionKey: selectionKey,
```

Comment:
nit: Rather than `{ selectionKey: selectionKey }` this can just be written as `{ selectionKey }` like is already being done for `containerFilter` and `containerPath`.